### ISSUE TITLE
Will not work

### ISSUE THREAD

**Issue by DJScripter** [griffpatch/griffpatch.github.io#13]
Whatever i try, never works.

**Comment by Barthdry** [comment]
No, This site is working perfectly https://griffpatch.github.io/scratch-gui/
Another thing is there is no tutorial so i managed to learn only few things. You can load these project there
